types/telebot/index.d.ts

```diff
@@ -347,7 +347,7 @@ declare class telebot {
 
     getChatAdministrators(chat_id: number | string): Promise<any[] | any>;
 
-    getChatMembersCount(chat_id: number | string): Promise<number>;
+    getChatMemberCount(chat_id: number | string): Promise<number>;
```

Comment:
The [telebot](https://github.com/mullwar/telebot) repository didn't change its api to reflect the official ones. In fact, the last commit was over a year ago. Please revert this line which caused CI builds failure. Thanks!